scrutiny.json

```diff
@@ -0,0 +1,82 @@
+                        "host_default": 8080,
+                        "label": "WebUI HTTP port",
+                        "ui": true
+                    }
+                },
+                "volumes": {
+                    "/config": {
+                        "description": "Choose a Share for Scrutiny config files. Eg: create a Share called Scrutiny-Share-Config for this purpose alone.",
+                        "label": "Config share"
+                    }
+                },
+                "environment": {
+                    "PUID": {
+                        "description": "Enter a valid system UserID for Scrutiny. Must have full permissions on the previous Share.",
+                        "label": "UID",
+                        "index": 1
+                    },
+                    "PGID": {
+                        "description": "Enter a valid system GroupID for Scrutiny. This group or the above UserID must have full permissions on the previous Share.",
+                        "label": "GID",
+                        "index": 2
+                    },
+                    "SCRUTINY_API_ENDPOINT": {
+                        "description": "API endpoint of the scrutiny UI. Do not change unless using as a remote collector. Default: 'http://localhost:8080'",
+                        "label": "SCRUTINY_API_ENDPOINT",
+                        "index": 3
+                    },
+                    "SCRUTINY_WEB": {
+                        "description": "Run the web service. Default: 'true'",
+                        "label": "SCRUTINY_WEB",
+                        "index": 4
+                    },
+                    "SCRUTINY_COLLECTOR": {
+                        "description": "Run the metrics collector. Default: 'true'",
+                        "label": "SCRUTINY_COLLECTOR",
+                        "index": 5
+                    }
+                }
+            }
+        },
+        "description": "<p><p><a href='https://github.com/AnalogJ/scrutiny'>Scrutiny</a> WebUI for smartd S.M.A.R.T monitoring. Scrutiny is a Hard Drive Health Dashboard &amp; Monitoring solution, merging manufacturer provided S.M.A.R.T metrics with real-world failure rates from Backblaze.</p></p><p><strong>Note:</strong>This image is updating the stats only once a day, if you see: '<i>No Devices Detected!</i>' after the installation, just wait 24 hours. <br>This process can also be triggered manually from the shell by executing the following command: <code>docker exec scrutiny scrutiny-collector-metrics run</code><br>More documentation on this can be found here: <a href='https://github.com/AnalogJ/scrutiny#usage'>Scrutiny GitHub</a></p>",
+		"more_info": "<p>Scrutiny is a Hard Drive Health Dashboard &amp; Monitoring solution, merging manufacturer provided S.M.A.R.T metrics with real-world failure rates from Backblaze.</p>",
+		"ui": {
+			"https": false,
+			"slug": ""
+		},
+		"volume_add_support": false,
```

Comment:
I assume, by adding the /dev:/dev volume, newly added devices after this Rock-on is installed will be automatically identified? I don't know enough about the whole device architecture at the lowest level so this might make no sense to enable the additional volume support flag.

Reply:
I think a simple "restart" of the container should bring all the new devices in. To test :smiley: 
On my knowledge, the way rockstor is currently managing the devices doesn't really allow to add new devices from the interface. But I could be wrong...

Reply:
Agreed on setting the `"volume_add_support"` to `false` here as it wouldn't help anyway. Indeed, this only allows to add volumes, but not devices (basically what @20design described) so it wouldn't allow users to "add devices post install".